terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java/org/terasoluna/gfw/common/codelist/i18n/SimpleI18nCodeList.java

```diff
@@ -335,4 +381,22 @@ public void setColumns(Map<String, Map<Locale, String>> cols) {
     public void afterPropertiesSet() {
         Assert.notNull(codeListTable, "codeListTable is not initialized!");
     }
+
+    /**
+     * Gets the default locale as fallback.<br>
+     * @return the default locale as fallback
+     */
+    public Locale getFallbackTo() {
+        return fallbackTo;
+    }
+
+    /**
+     * Sets the default locale as fallback.<br>
+     * Defaults to {@link Locale#getDefault()}
+     * @param fallbackTo the default locale as fallback
+     */
+    public void setFallbackTo(Locale fallbackTo) {
+        Assert.notNull(fallbackTo, "fallbackTo must not be null");
+        this.fallbackTo = fallbackTo;
+    }
```

Comment:
- [ ] Please add JavaDoc description about resolving locale. (class level, and resolving method level).

Reply:
Modified by a94a857